worlds/shorthike/Options.py

```diff
@@ -61,6 +61,21 @@ class CostMultiplier(Range):
     range_end = 200
     default = 100
 
+class FillerCoinAmount(Choice):
+    """The amount of coins that will be in each filler coin item."""
+    display_name = "Coins per Filler Item"
+    option_7_coins = 0
```

Comment:
I haven't played the game, so maybe these make sense in context, but these numbers seem very strange. What are they from/for?

Reply:
The numbers are the coin amounts currently in the item pool. Those come from the vanilla amounts of coins that you obtain in different locations.